tests/model_fixtures.py

```diff
@@ -0,0 +1,93 @@
+"""
+This is where we put all the data about creating / updating models
+"""
+
+from dataclasses import dataclass
+from enum import Enum
+from typing import Dict, Sequence, Union
+
+from pangeo_forge_orchestrator.model_builders import MultipleModels
+from pangeo_forge_orchestrator.models import MODELS
+
+# Model fixture containers --------------------------------------------------------------
+
+
+class APIErrors(str, Enum):
+    """Error types that may occur if field input does not conform to the schema defined by the
+    table's Pydantic base model.
+    """
+
+    datetime = "datetime"
+    enum = "enum"
+    int = "integer"
+    missing = "missing"
+    str = "string"
+
+
+APIOpts = Dict[str, Union[int, str]]
+
+
+@dataclass
+class ModelFixtures:
+    model: MultipleModels
+    required_fields: Sequence[str]
+    create_opts: Sequence[APIOpts]
+    invalid_opts: Sequence[APIOpts]
+    update_opts: Sequence[APIOpts]
+
+
+NOT_STR = {"not parsable": "to str"}
+NOT_INT = "not parsable to int"
+NOT_ISO8601 = "Jan 01 2021 00:00:00"
+
+
+recipe_run_fixtures = ModelFixtures(
+    model=MODELS["recipe_run"],
+    required_fields=[
+        "recipe_id",
+        "bakery_id",
+        "feedstock_id",
+        "head_sha",
+        "version",
+        "started_at",
+        "status",
```

Comment:
@cisaacstern - these tests have revealed that `status` is not, in fact, a required option, since it has a default value. Ok if I remove it from the required fields?

Reply:
Good catch! Yep 👍